inferentia/qa/setup_test_enviroment_and_test.sh

```diff
@@ -109,6 +122,7 @@ docker create --name ${CONTAINER_NAME}             \
             -e TEST_REPO=${TEST_REPO}              \
             -e TEST_JSON_REPO=${TEST_JSON_REPO}    \
             -e TRITON_PATH=${TRITON_PATH}          \
+            -e USE_TENSORFLOW="1"                  \
```

Comment:
Don't we want to add test for TF version 2 as well?

Reply:
No there's something that doesn't work for tf2. Since we don't need it for MLPerf right now I think we can just stick with testing with TF1